### DISCUSSION TITLE
Please consider adding an upload button for uploading images on mobile

### DISCUSSION THREAD

**Discussion by kjgcoop** [karakeep-app/karakeep#328] [Ideas]
Thanks for creating an awesome app! I see I can drag and drop images on the computer, but it would be nice to upload images from my phone.

Thank for your consideration!

**Comment by MohamedBassem** [comment]
Hey, you can actually upload images from the phone by clicking the share button on the photo and then choosing hoarder as the sharing destination

**Comment by kjgcoop** [comment]
Thanks for your quick reply! I didn't even realize there was an Android app, that's awesome!

Thank you!
